### DISCUSSION TITLE
[help] brew tries to use proxy when no proxy env var exists

### DISCUSSION THREAD

**Discussion by tddschn** [Homebrew/discussions#2180] [Everyday usage] [answered]
I was not using any proxy env vars, but it seems that brew tries to use one:
```
HOMEBREW_CURL_VERBOSE=1 brew update
fatal: unable to access 'https://github.com/Homebrew/brew/': Failed to connect to 127.0.0.1 port 8888: Connection refused
...
```
Is there anything I'm missing that causes brew to use a proxy?

TIA.

**Comment by SMillerDev** [comment, marked as answer]
It doesn't have to be a proxy. It could also be ssh, dns or git settings. See what IP you get when you `ping github.com`

**Reply by tddschn** [reply]
Thank you! It was the `http` proxy setting in my global git config. :)
